index.bs

```diff
@@ -466,15 +521,13 @@ spec: webrtc-pc; urlPrefix: https://www.w3.org/TR/webrtc/#
           return a Promise rejected with a <code>TypeError</code>.
         </li>
         <li>
-          If <code>configuration.video</code> in non-null and
-          <code>configuration.video.contentType</code> is not a <a>valid
-          VideoConfiguration contentType</a>, return a Promise rejected with a
+          If <code>configuration.video</code> is <a>present</a> and is not a
```

Comment:
The intent is that malformed mime types wouldn't be a valid MIME types. The spec around MIME types is a huge spaghetti so I have to admit I'm not sure if it requires the parameters to be valid or if invalid parameters would be ignored. This also triggers a question with regards to what we should do. OOC, what does implementations would do (for say isTypeSupported) if the parameter is malformed?